### ISSUE TITLE
Saving

### ISSUE THREAD

**Issue by ethan-w-foshee** [ethan-w-foshee/TerraStella#8]
Saving, as mentioned in [this PR](https://github.com/ethan-w-foshee/TerraStella/pull/7), should be really easy. Just using local storage to store the main JSON object, yeah?

**Comment by CyborgPotato** [comment]
Yes indeedy, just make sure all relevant information is stored in at least one JSON

**Comment by ethan-w-foshee** [comment]
Should we have it all be in the main.js? Or could we store it in a different file easily? Even still just using a static html file?

**Comment by CyborgPotato** [comment]
We can p easily have things in different files, though I'm debating we might have some kind of build system (with node or webpack etc.) so that when deploying it combines it all into one file for consumption

**Comment by ethan-w-foshee** [comment]
While we could, right now, I don't see a point in having it that way.

But I do think saving it all in a different file would be good.
